configuration.nix

```diff
@@ -0,0 +1,53 @@
+{ config, pkgs, lib, ... }:
+
+{
+
+  nixpkgs.config =
+    {
```

Comment:
This `{` is oddly indented compared to others (see below).

Reply:
Dealt with in latest push.